### DISCUSSION TITLE
Create a Hurl Entry from a Curl?

### DISCUSSION THREAD

**Discussion by leeola** [Orange-OpenSource/hurl#278] [Ideas] [answered]
To rapidly create tests it would be helpful to be able to create Hurl Entries from a Curl command directly. Eg allowing a workflow of going through a browser application and copying the API requests via `Copy as cURL` and importing that directly into Hurl.

Is that possible now?

**Comment by jcamiel** [comment]
Hmm, it’s not possible now but that’s a nice idea! We could do a form on https://hurl.dev that display the conversion

**Comment by fabricereix** [comment]
It's indeed very useful to convert sessions between different formats.
We already thought about converting from an HAR exported from the browser to a Hurl file.

For a first step, we could write simple python scripts for these conversions.

**Comment by jcamiel** [comment]
@leeola I've transformed this discussion to an issue #316  as we're going to only use issues

**Comment by jcamiel** [comment, marked as answer]
As of 3.0.0, we can use `hurlfmt` to convert curl commands to Hurl file.
